### ISSUE TITLE
Way to get rid of console

### ISSUE THREAD

**Issue by dinhanhx** [cody-bui/Echo#2]
![image](https://user-images.githubusercontent.com/38489776/73809865-28850a80-4807-11ea-9f38-635ae155123a.png)
I assume you are using Visual Studio. 
Just open your project, click to Debug then YourProjectName Properties then Linker then System then SubSystem then choose Windows (/SUBSYSTEM:WINDOWS)

**Comment by cody-bui** [comment]
Yes, I'm fully aware of the issue, but I haven't updated the project generation script and source code to run as console program or windowed program yet
